WordPress/Classes/ViewRelated/Comments/FullScreenCommentReplyViewController.swift

```diff
@@ -0,0 +1,152 @@
+
+import UIKit
+import Gridicons
+
+/// FullscreenCommentReplyViewController is used when full screen commenting
+/// After instantiating using `newEdit()` the class expects the `content` and `onExitFullscreen`
+/// properties to be set.
+
+class FullScreenCommentReplyViewController: EditCommentViewController {
```

Comment:
Another very good candidate for some fresh unit tests! 💯

Reply:
I honestly thought very hard about this but I honestly wasn't really sure what I would test on this view. I'm open for a learning experience on what types of things you see to be tested here?

Reply:
@leandroalonso and I continued this conversation in Slack and came up with a good idea for testing which I've implemented below.